installer/win/WixSharpInstaller/Symphony.cs

```diff
@@ -35,7 +35,7 @@ static public void Main(string[] args)
         // desired contents of installation, and then we can simplify this bit.
         var project = new ManagedProject(productName,
             new Dir(@"%ProgramFiles%\" + productName,
-                new File(new Id("com.symphony.electron_desktop"), @"..\..\..\dist\win-unpacked\Symphony.exe",
+                new File(new Id("symphony_exe"), @"..\..\..\dist\win-unpacked\Symphony.exe",
```

Comment:
Do we still need to revert app Id?

Reply:
Not required, Should I update to the new one?

Reply:
if it has no impact on auto-update then no need to update it to the new one.